### PR TITLE
refactor(form): apply Vaadin ESLint rules

### PR DESCRIPTION
**Pull request by platosha**
Depends on #1130

### CODE REVIEW

File: packages/ts/form/src/Field.ts [hunk 1]
```diff
@@ -1,13 +1,16 @@
+/* eslint-disable accessor-pairs,sort-keys */
 import { type ElementPart, noChange, nothing, type PropertyPart } from 'lit';
 import { directive, Directive, type DirectiveParameters, type PartInfo, PartType } from 'lit/directive.js';
+import { BooleanModel } from '../Models';
```
Comment (cromoteca) [inline packages/ts/form/src/Field.ts:4]:
```suggestion
import { BooleanModel } from '../Models.js';
```

**Review by cromoteca — CHANGES_REQUESTED** [review]

**Comment by Lodin** [comment]
Have you decided to postpone moving to ES native private members? I see several places where you could do that safely in the code.

File: packages/ts/form/src/Validators.ts [hunk 2]
```diff
@@ -310,7 +310,7 @@ export class Digits<T> extends AbstractValidator<T> {
 
   override validate(value: T): boolean {
     return (
-      String(toFloat(String(value))).replace(/(.*)\.\d+/u, '$1').length <= this.integer &&
+      String(toFloat(String(value))).indexOf('.') <= this.integer &&
```
Comment (cromoteca) [inline packages/ts/form/src/Validators.ts:313]: This change seems OK to me, anyway I question both implementation about negative number, as the minus sign seems to be taken into account here. 

There are no tests for negative numbers indeed.

I'm going to open an issue for that.

Comment (Lodin) [reply packages/ts/form/src/Validators.ts:313]: I believe that we shouldn't do such kind of changes in this PR. Could we ignore the Sonar quality gates and create an issue for these issues?

Comment (cromoteca) [reply packages/ts/form/src/Validators.ts:313]: Except that it's blocking merging

Comment (cromoteca) [reply packages/ts/form/src/Validators.ts:313]: Concerning the minus sign yes, I opened a different issue.

Comment (Lodin) [reply packages/ts/form/src/Validators.ts:313]: Hmm, I was sure we could ignore Sonar if it's better for us 🤔

Comment (Lodin) [reply packages/ts/form/src/Validators.ts:313]: @platosha what do you think?

Comment (platosha) [reply packages/ts/form/src/Validators.ts:313]: let's address in a separate PR

Comment (platosha) [reply packages/ts/form/src/Validators.ts:313]: Sonar is red but not blocking I think

**Review by Lodin — DISMISSED** [review]

File: packages/ts/form/src/Validators.ts (hunk 2 above)
Comment (cromoteca) [reply packages/ts/form/src/Validators.ts:313]: OK so I'm going to revert my changes.

**Review by cromoteca — APPROVED** [review]

**Pull request merged**